.github/.dependabot

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
I feel that this might be too aggressive and cause a lot of noise -- what do you think of weekly / monthly?